src/views/MarketStatsDetails.tsx

```diff
@@ -249,12 +250,28 @@ const DetailsItem = ({
     }
     case MarketStats.Funding1H: {
       return (
-        <$Output
-          type={OutputType.Percent}
-          value={value}
-          color={color}
-          fractionDigits={TINY_PERCENT_DECIMALS}
-        />
+        <WithTooltip
+          slotTooltip={
+            <dl>
+              <dd tw="flex">
+                {stringGetter({ key: STRING_KEYS.ANNUALIZED })}:
+                <Output
+                  tw="ml-0.25"
+                  type={OutputType.Percent}
+                  value={MustBigNumber(value).times(24 * 365)}
+                  fractionDigits={0}
+                />
```

Comment:
would be cool if we either added color or colored sign

Reply:
I think it's fine since you get color from the thing you're hovering over